content/concepts/consensus-network/amendments/contribute-code-flow.md

```diff
@@ -0,0 +1,82 @@
+---
+html: contribute-code-flow.html
+parent: amendments.html
+blurb: 
+labels:
+  - Blockchain
+---
+# Contribute Code to the XRP Ledger
+
+The software that powers the XRP Ledger is open-source, so anyone can download, modify, extend, or explore it. If you want to contribute code, it's important to work with the community in defining the specifications of your changes and testing the code before adding it to `rippled`.
+
+
+## Standards Draft
+
+Most changes will start with a standards draft. The standards draft outlines the specifications of your change and provides the XRPL community a chance to give feedback. Before committing to development, always submit a standards draft to the [XRPL-Standards repo](https://github.com/XRPLF/XRPL-Standards/discussions/categories/standard-proposals).
+
+**Note:** Bug fixes don't require standards drafts, but may require amendments.
+
+Your standards draft should supply the community with information to facilitate useful discussion. Helpful sections to include are:
+
+- Abstract/Introduction
+- Specifications
+- Considerations
+
+***TODO: Link to a standards draft outline.***
+
+After receiving sufficient input from the community, you can move forward in the process. "Sufficient" is subjective in open-source development, but gauging the community response to your standards draft can help you understand how it will be received if it needs to be voted on as an amendment.
+
+
+## Amendment Implementation
+
+After you've finalized the standards draft with the community, you now need to determine if your change requires an amendment. Anything that affects **transaction processing** requires an amendment. This applies to:
+
+- Features
+- Enhancements
+- Bug Fixes
+
+**Note:** If your change doesn't affect how transactions are processed by the ledger, you can go straight to coding and deployment.
```

Comment:
Explain, e.g. 'API change' (?)

Reply:
I think if we get the general pointers of what requires an amendment down, it'd be better to have a couple examples to show here. Something that might trip someone up, so like an API change that modifies a transaction, but not in a way that it changes how the transaction is processed. Good examples to include should illustrate that difference between changes and changes that affect ledger outcomes.

Did you have a couple off the top of your head? If not I can go look through past PRs.